src/main/res/layout/account_selection_list_item.xml

```diff
@@ -60,16 +60,17 @@
                       android:layout_width="wrap_content"
                       android:layout_height="wrap_content">
 
-            <TextView android:id="@+id/addr"
-                      android:layout_width="wrap_content"
-                      android:layout_height="wrap_content"
-                      android:textDirection="ltr"
-                      android:singleLine="true"
-                      android:ellipsize="marquee"
-                      android:textAppearance="?android:attr/textAppearanceSmall"
-                      android:textSize="14sp"
-                      android:fontFamily="sans-serif-light"
-                      tools:text="user@example.com" />
+          <org.thoughtcrime.securesms.components.emoji.EmojiTextView
+              android:id="@+id/addr"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:textDirection="ltr"
+              android:singleLine="true"
+              android:ellipsize="marquee"
+              android:textAppearance="?android:attr/textAppearanceSmall"
+              android:textSize="14sp"
+              android:fontFamily="sans-serif-light"
+              tools:text="user@example.com" />
```

Comment:
ah, this is the old "address" :)

maybe we should rename it to sth. more fitting.

also, if no label is set, the alignment is not centered vertically:

<img width=300 src=https://github.com/user-attachments/assets/9ee55c31-f510-44f6-ab48-51a09eaf1e56>

Reply:
fixed